lib/src/trim_editor.dart

```diff
@@ -516,4 +520,13 @@ class _TrimEditorState extends State<TrimEditor> with TickerProviderStateMixin {
       ),
     );
   }
+
+  _refreshVideoIfNeeded() async {
+    if (Platform.isIOS) {
+      await videoPlayerController.play();
```

Comment:
Last question below wont work: ?
```
await videoPlayerController.play()
awair videoPlayerController.pause()
```

Reply:
Nope, that was also my first try. Unfortunately, it won't work.